src/Main.elm

```diff
@@ -0,0 +1,46 @@
+module Main exposing (main)
+
+import Browser
+import Html
+
+
+main : Program () Model Msg
+main =
+    Browser.document
```

Comment:
Guessing Browser.application is too much for now?

Reply:
Yeah - I thought this was a good first step. We might end up embedding in their wordpress - so I toyed with using element even... but this is simpler as a standalone for now.